swarm/api/api.go

```diff
@@ -355,10 +357,10 @@ func (a *API) Get(ctx context.Context, manifestAddr storage.Address, path string
 
 			// if it's multihash, we will transparently serve the content this multihash points to
 			// \TODO this resolve is rather expensive all in all, review to see if it can be achieved cheaper
-			if rsrc.Multihash {
+			if rsrc.Multihash() {
```

Comment:
It's not shown here but there is a method for `resourceUpdate` declared in `resource.go`, it should be in the file where `resourceUpdate` is declared.

Reply:
Fixed, thanks.